packages/orbit-components/src/InputGroup/InputGroup.stories.tsx

```diff
@@ -70,20 +70,30 @@ const selectOptionsMonths = [
 ];
 
 export const DateOfBirth: Story = {
-  render: ({ error, help, selectValue, ...args }) => (
-    <InputGroup {...args}>
-      <InputField
-        placeholder="DD"
-        error={error}
-        help={help}
-        onBlur={action("onBlurIn")}
-        onFocus={action("onFocusIn")}
-        onChange={action("onChangeIn")}
-      />
-      <Select options={selectOptionsMonths} value={selectValue} placeholder="Month" />
-      <InputField placeholder="YYYY" />
-    </InputGroup>
-  ),
+  render: ({ error, help, selectValue, ...args }) => {
+    const [value, setValue] = React.useState(selectValue);
```

Comment:
This is a story. This will never happen. The change is unnecessary.

Reply:
Without the `useEffect` implementation, value in `Select` is not changed in case I pick a different value in _Controls_ panel:

![image](https://github.com/user-attachments/assets/d82cf927-0185-4251-8b28-7fdafdb0ac0d)

Reply:
Hmm I see, you can leave it then